api/orchestrator/orchestrator.proto

```diff
@@ -655,16 +655,20 @@ message CloudService {
     (validate.rules).repeated .items.message.required = true
   ];
 
-  // As long as repeated maps are not supported, we have to do this workaround. 
-  message Tag {
-    map<string, string> tag   = 1;
+  message Metadata {
```

Comment:
A "created_at" property would also be good, so that we know when the cloud service was created. Not sure if this would be part of the metadata property or directly a property of the service.